packages/firestore/test/unit/local/persistence_test_helpers.ts

```diff
@@ -93,11 +93,11 @@ export async function testIndexedDbPersistence(
   options: {
     dontPurgeData?: boolean;
     synchronizeTabs?: boolean;
-  } = {}
+  } = {},
+  prefix = `${TEST_PERSISTENCE_PREFIX}/`
```

Comment:
Nit: What do you think of this signature?

```
 testIndexedDbPersistence(
  options: {	  options: {
    dontPurgeData?: boolean;	   
    synchronizeTabs?: boolean;	  
    prefix?: string (or ` = ${TEST_PERSISTENCE_PREFIX}/`)
  } = {}} = {},
```

Just my personal preference, so feel free to ignore.

Reply:
Reverted, see below.